source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3305,9 +3280,8 @@ export class Choice extends SerializableObject {
         Versions.v1_0,
         "inlineAction",
         [
-            "Action.ShowCard",
-            "Action.Submit",
-            "Action.OpenUrl"
+            "*",
+            "-Action.ToggleVisibility"
```

Comment:
Sorry if I was unclear - this won't just work. It's the right syntax, but it is not currently supported. It would have to be implemented in `SerializationContext.internalParseCardObject`, which might be a bit outside the scope of this PR. Feel free to revert to what you had before for now, but do include Action.Execute in the list.